spec/rubocop/cop/lint/shared_mutable_default_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::SharedMutableDefault, :config do
+  context 'when line is unrelated' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        puts 'hi'
```

Comment:
Is this line necessary?

Reply:
No! We just always start off with ensuring it doesn't flag completely unrelated code in our internal style.